assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/datesListTableRow.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { format } from 'date-fns';
+import { __ } from '@wordpress/i18n';
+
+import DateRegistrationsLink from '@edtrUI/datetimes/DateRegistrationsLink';
+import DateActionsMenu from '@edtrUI/datetimes/datesList/actionsMenu/DateActionsMenu';
+import { Datetime } from '../../../../services/apollo/types';
+import { DisplayDates } from '../../../../interfaces/datetimes/types';
+import { getBackgroundColorClassName, status } from '@sharedEntities/datetimes/helpers';
+import { InlineEditText } from '@appInputs/InlineEditInput';
+import { shortenGuid } from '@appServices/utilities/text';
+import { useDatetimeMutator } from '@edtrServices/apollo/mutations';
+
+import '../../../../../../application/ui/styles/root/entity-status.css';
+
+const DATE_TIME_FORMAT = 'EEE MMM yy h:mm a';
+
+interface Props {
+	datetime: Datetime;
+	displayDates: DisplayDates;
+}
+
+/**
+ * EditorDateEntityListItem
+ * Displays Event Date as a table row similar to existing eventEntity editor UI
+ */
+const datesListTableRow = ({ datetime, displayDates }: Props) => {
+	const bgClassName = getBackgroundColorClassName(datetime);
+	const id = datetime.dbId || shortenGuid(datetime.id);
+	const statusClassName = status(datetime);
+	const { updateEntity } = useDatetimeMutator(datetime.id);
+
+	const capacity = {
+		key: 'capacity',
+		type: 'cell',
+		className: 'ee-date-list-cell ee-date-list-col-capacity ee-rspnsv-table-column-tiny ee-number-column',
+		value: (
+			<InlineEditText
+				className={'ee-focus-priority-5'}
```

Comment:
thnx for adopting these 👍

Reply:
it can be improved further with a mapped object or enums